File: Python Basics/conditional_stetaments/area_of_figures.py
import math

figure = input()
a = float(input())

if figure == "square":
    area = a ** 2
elif figure == "rectangle":
    b = float(input())
    area = a * b
elif figure == "circle":
    area = math.pi * a ** 2
elif figure == "triangle":
    h = float(input())
    area = a * h / 2

print(f'{area:.3f}')

File: JS/Conditional Statements/AreaOfFigures.js
function figureAreaCalc(params) {
    let figure = params[0];
    if (figure === "square") {
        let area = (Number(params[1]) ** 2).toFixed(3);
        console.log(area);
    } else if (figure === "rectangle") {
        let area = (Number(params[1]) * Number(params[2])).toFixed(3);
        console.log(area);
    } else if (figure === "circle") {
        let area = (Math.PI * Number(params[1]) ** 2).toFixed(3);
        console.log(area);
    } else if (figure === "triangle") {
        let area = (Number(params[1]) * Number(params[2]) / 2).toFixed(3);
        console.log(area);
    }
}